### DISCUSSION TITLE
Storage Limit ??

### DISCUSSION THREAD

**Discussion by am185760** [Blazored/LocalStorage#218] [Q&A]
Hi Blazored.LocalStorage team,

Please share the capacity of this storage. I got a list<Model> of 20k, and store command not working..

**Comment by chrissainty** [comment]
The storage limit is controlled by the browser not by this library. 

https://developer.mozilla.org/en-US/docs/Web/API/Storage_API/Storage_quotas_and_eviction_criteria
